### PR TITLE
Animal Table

### PR DESCRIPTION
**Pull request by braincee**
- I created a `vet_clinic` database.
- I created a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
- I inserted into the table with the following data:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

- I wrote queries with the following quests:
 - Find all animals whose name ends in "mon".
 - List the name of all animals born between 2016 and 2019.
 - List the name of all animals that are neutered and have less than 3 escape attempts.
     
    
Screenshot (119)

   
 - List the date of birth of all animals named either "Agumon" or "Pikachu".
 - List name and escape attempts of animals that weigh more than 10.5kg
 - Find all animals that are neutered.
 - Find all animals not named Gabumon.
 
     
   
Screenshot (120)

 - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 
   10.4kg 
 or 17.3kg)
 

Screenshot (121)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = True AND escape_attempts < 3;
```
Comment (Cathella) [inline queries.sql:7]: [OPTIONAL] Kindly consider putting values in quotes as seen below

```
SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = True;
+
+SELECT * FROM animals WHERE name != 'Gabumon';
```
Comment (Cathella) [inline queries.sql:15]: [OPTIONAL] This can also be expressed as below
```
SELECT * FROM animals WHERE name <> 'Gabumon';
```

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @braincee 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Professional README.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @Cathella 

**_Note: Please I have implemented all the required and optional changes._**

mousse-nena

**Pull request merged**